extensions/wikia/Wall/sql/wall_history_local.sql

```diff
@@ -1,7 +1,6 @@
 CREATE TABLE wall_history (
 	parent_page_id INT(8) UNSIGNED,
 	post_user_id INT(10) UNSIGNED,
-	post_user_ip INT UNSIGNED,
```

Comment:
Let's add a drop of `post_user_ip` column to update.php.

Reply:
I've updated `WikiaUpdater`.